src/common/errorAlert.js

```diff
@@ -0,0 +1,10 @@
+import { Alert } from 'react-native';
+
+export const showErrorAlert = (message, title) => {
+  Alert.alert(
+           title,
+           message,
+           [{ text: 'OK', onPress: () => {} }],
+           { 'cancelable': true }
```

Comment:
cancelable is for the when the user presses outside the alert box, so if a user presses outside of the alert then it can be dismissed!